src/WebdriverClassicDriver.php

```diff
@@ -937,12 +937,24 @@ private function applyTimeouts(): void
                     case 'script':
                         $timeouts->setScriptTimeout($param / 1000);
                         break;
+
                     case 'implicit':
                         $timeouts->implicitlyWait($param / 1000);
                         break;
+
+                    case 'page load':
+                    case 'pageLoad':
+                        @trigger_error(
+                            "Using \"$type\" timeout type is deprecated, please use \"page\" instead",
+                            E_USER_DEPRECATED
+                        );
```

Comment:
Where are the tests for this code?

Reply:
I did give it a try and gave up after finding that PHPUnit has deprecated...catching deprecations. 😅
https://phpunit.de/announcements/phpunit-10.html#:~:text=Test%20Outcome%20versus%20Test%20Issues

Any idea how to continue further? The only way I see is by registering our own `set_error_handler` 🙄. Although I remember there was something custom in the driver testsuite about this and symfony. 🤔

Reply:
Next steps:

1. merge the #39 to `main` branch
2. rebase this PR upon the `main` branch
3. adjust the `testShortPageLoadTimeoutThrowsException` test (was added in #39) to support data provider (as was done in https://github.com/minkphp/MinkSelenium2Driver/pull/400) and when `$timeoutType` would be `page load` or `pageLoad` we'll see how current error handler (including Symfony's version of it) will work on PHPUnit 9 and PHPUnit 10

Reply:
I've rebased this PR on top of `main` branch after #39 was merged. Now you can experiment with `testShortPageLoadTimeoutThrowsException`.

Reply:
PHPUnit 11 has a way to write assertions on deprecations. And the Symfony PHPUnit bridge has a trait that we can use in test cases to write assertions for deprecations using the same API.

Reply:
Do you mean `Symfony\Bridge\PhpUnit\ExpectDeprecationTrait`, which provides the `expectDeprecation` method?

But due to unknown reasons, an attempt to set timeouts in combination with deprecation checking causes issues for window test https://github.com/minkphp/webdriver-classic-driver/actions/runs/11662801115/job/32469893182 .